src/sentry/api/endpoints/debug_files.py

```diff
@@ -80,6 +82,67 @@ def has_download_permission(request, project):
     return roles.get(current_role).priority >= roles.get(required_role).priority
 
 
+@region_silo_endpoint
+class ProguardArtifactReleasesEndpoint(ProjectEndpoint):
+    permission_classes = (ProjectReleasePermission,)
+
+    def post(self, request: Request, project) -> Response:
+        release_name = request.data.get("release_name")
+        proguard_uuid = request.data.get("proguard_uuid")
```

Comment:
Can we somehow do some validation of the incoming data? In case an invalid UUID is passed a database error will be thrown which is not very nice from the user's perspective.